4.0.0/alpine/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM alpine:3.6
+MAINTAINER "Philip O'Toole <philip.otoole@yahoo.com>"
+
+# Here we install GNU libc (aka glibc) and set C.UTF-8 locale as default.
+# required to use 'rqlited' within alpine as it is not statically linked
+RUN ALPINE_GLIBC_BASE_URL="https://github.com/sgerrand/alpine-pkg-glibc/releases/download" && \
+    ALPINE_GLIBC_PACKAGE_VERSION="2.23-r3" && \
+    ALPINE_GLIBC_BASE_PACKAGE_FILENAME="glibc-$ALPINE_GLIBC_PACKAGE_VERSION.apk" && \
+    ALPINE_GLIBC_BIN_PACKAGE_FILENAME="glibc-bin-$ALPINE_GLIBC_PACKAGE_VERSION.apk" && \
+    ALPINE_GLIBC_I18N_PACKAGE_FILENAME="glibc-i18n-$ALPINE_GLIBC_PACKAGE_VERSION.apk" && \
+    apk add --no-cache --virtual=.build-dependencies wget ca-certificates && \
+    wget \
+        "https://raw.githubusercontent.com/andyshinn/alpine-pkg-glibc/master/sgerrand.rsa.pub" \
```

Comment:
What is the source of this cert, key, library, etc? I don't want to publish Docker images unless I understand the source of the glibc inside.